convert_val

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+#
+#                         License
+#
+# Copyright (C) 2024  David Valin dvalin@redhat.com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Convert various time and memory units.
+
+usage()
+{
+	echo "usage $1 <unit>"
+	echo "-h --help --usage: This help message"
+	echo "--to_unit <u>: Unit to convert to."
+	echo "--value <value>: Value to be converted."
+	echo "--from_unit <u>: Unit of value to be converted."
+	echo "--time_val: Units being worked with are time vals"
+	echo "If just --from_unit is passed, program returns the unit value.  EG.  --from_unit K will return 1000B"
+	echo "If --value and --from_unit, program returns the value in the base unit."
```

Comment:
Specify the base units, what are they?

Reply:
I think I wasn't clear, if `--value` and `--from_unit` are specified, what unit does it return?  What about if `--time_val` is specified, does that change anything?

Reply:
Listing out all the scaling units is a bit excessive

Reply:
So what you are looking for is the defaults, right?

Reply:
Essentially yes, if `--to_unit` is not specified, what unit does it convert to?

Reply:
Updates as requested made.